src/modules/visits/types.ts

```diff
@@ -29,6 +29,11 @@ export enum VisitTimeframe {
 	FUTURE = 'FUTURE',
 }
 
+export enum VisitAccessType {
+	FULL = 'FULL',
+	FOLDERS = 'FOLDERS',
+}
```

Comment:
You can export a renamed version of the generated database enum. That way they always stay in sync with the database enum lookup table:
```suggestion
export { Lookup_Maintainer_Visitor_Space_Request_Access_Type_Enum as VisitAccessType } from '~generated/graphql-db-types-hetarchief';
```